config/locales/diaspora/en.yml

```diff
@@ -1039,6 +1039,13 @@ en:
       reshare_by: "Reshare by %{author}"
 
   report:
+    modal:
+      title: "You are about to send an email to all %{pod_name} moderators"
+      report_only_if_violated_terms: "You should only report posts or comments which violates"
+      terms_link: "%{pod_name} terms"
+      you_can_blocK: "If the content isn't forbidden but you don't want to see it, you can block the authors from their profile page. If the content is a comment on your own post, you can delete it directly."
```

Comment:
>My understanding is that ignoring doesn't prevent you from seeing comments on other people's posts; it just stops you from seeing that person's own posts, and stops them from interacting on your posts.

Yeah I just made the test and indeed the comment of an ignored user is still visible on someone else post. So I'm not sure about "you can hide it", as it is only true in some cases.

Reply:
Is this a bug or a feature? I would expect not to see anything if I ignore the user. Can you create an issue then?

Reply:
I can't find of anything better right now, so I'll go with "If it doesn't but you don't want to see it, you can ignore the authors from their profile page. If the content is a comment on your own post, you can delete it directly" as the sentence is just after "Reporting content sends an email to all the moderators of this pod. Please only report content that violates pod_name terms" I think there is no need to repeat "violates ..."

Reply:
You could replace the first sentence with "If you want to stop someone interacting on your posts, you can ignore them from their profile page."

It doesn't say it will hide all content, but gives an action you can do.

Reply:
> Is this a bug or a feature?

@tclaus the ignore feature is not meant to be like Facebook's 'block' in which it's as if the other person no longer exists. It's called ignore because it's primarily a method of curating your stream; it means that their posts will no longer appear in your stream even if they contain tags you follow. It's primarily intended to deal with people who post a lot of stuff you're not interested in reading rather than harassment. That's why we should continue to refer to it as 'ignore' rather than 'block'.

Reply:
Oh yeah, we did the test one month ago in that exact thread :sweat_smile: